### ISSUE TITLE
Broken tint package when building for x230

### ISSUE THREAD

**Issue by kedare** [Thrilleratplay/coreboot-builder-scripts#16]
When running the script for x230, the build process stop at this step :

```
--2021-08-20 17:48:17--  https://mirror.fsf.org/trisquel/pool/main/t/tint/tint_0.04+nmu1.tar.gz
Resolving mirror.fsf.org (mirror.fsf.org)... 209.51.188.52, 2001:470:142:4::c
Connecting to mirror.fsf.org (mirror.fsf.org)|209.51.188.52|:443... connected.
HTTP request sent, awaiting response... 404 Not Found             
2021-08-20 17:48:18 ERROR 404: Not Found.
                                                                                                                                    
tar: tint_0.04+nmu1.tar.gz: Cannot open: No such file or directory 
tar: Error is not recoverable: exiting now                                                                                          
rm: cannot remove 'tint_0.04+nmu1.tar.gz': No such file or directory             
mv: cannot stat 'tint-0.04+nmu1': No such file or directory
make[1]: *** [Makefile:26: download] Error 1                                                                                        
make: *** [payloads/external/Makefile.inc:210: payloads/external/tint/tint/tint.elf] Error 2
Uh oh. Things did not go according to plan.   
```

The required file is apparently missing from the url.

**Comment by Thrilleratplay** [comment]
Not something I would have expected to break...but...it seems like it was corrected in [coreboot patchset #48616]( https://review.coreboot.org/c/coreboot/+/48616).  If you are trying to build an older version of coreboot, the cherry picked patch may need to be added to `x230/compile.sh` after `downloadOrUpdateCoreboot`

**Comment by kedare** [comment]
Hmm I did not specify any version during the build, so it theory that should not have happened ?

**Comment by Thrilleratplay** [comment]
@kedare There are a few ways this could happen and setting an older version is just the most obvious.  The build scripts were written to automate and assist but not be overly intrusive.  What is the full command line you used for `build.sh`?  Have you used these scripts before or did you just clone/download the repo?  If you don't know, tint this is just a secondary payload that is a Tetris clone, if it is something you will never use it can be removed.

**Comment by Thrilleratplay** [comment]
Verified this was fixed in coreboot v4.21. However, as the sha256 release file was not updated, either `--bleeding-edge` or `-t 4.21` flags will be needed when using these build scripts.  Hopefully this can be corrected soon.
